brave/brave-core
# Updated ecosia brand image

fix https://github.com/brave/brave-browser/issues/23405


Resolves 

![Screenshot_20220613-231630](https://user-images.githubusercontent.com/6786187/173507203-a4a4c65d-1d36-481f-86b8-2ba30a16472b.png)


## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Set android region that ecosia is in default provider list (See below note for ecosia regions)
2. Do fresh install and launch and first touch to omnibox shows search engine on boarding page
3. Check ecosia uses latest brand logo

Note:
According to the source code(https://github.com/brave/brave-core/blob/master/chromium_src/components/search_engines/template_url_prepopulate_data.cc#L91),
below regions code include ecosia as one of the default provider.
* AU(austrailia), IE(ireland), AT(austria), BE(belgium) and etc...
